README.md

```diff
@@ -83,6 +83,17 @@ with only a subset of sections having recommendations):
   * Load-balancing
   * [Failure Handling](./docs/operations/failurehandling.md)
 
+## Module Diligence
+
+As a minium, any modules recommended as part of this reference architeucture should meet the following criteria:
+
+* The module is not deprecated.
+* The module has an appropriate license.
+  * _Can we define what we consider an 'appropriate' license is in this context?_
```

Comment:
I think we could define a set of licences that are generally acceptable (MIT, Apache 2 etc.) and those that are not in the list should be flagged for additional review/agreement that it still makes sense to recommend the module.